### PR TITLE
[PL] Translation of documentation lesson

### PR DESCRIPTION
**Pull request by Koziolek**
In scope of issue #202

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @Koziolek!  cc @kdebowski for review

File: pl/lessons/basics/documentation.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+  - `#` - Służy do komentowania kodu.
+  - `@moduledoc` - Pozwala na tworzenie dokumentacji modułów.
+  - `@doc` - Pozwala na tworzenie dokumentacji poszczególnych funkcji.
+
+### Komentowanie kodu
+
+Najprostszą metodą dokumentowania kodu są komentarze. Podobnie jak w Pythonie i Ruby, tak w Elixirze komentarz 
+rozpoczyna się znakiem `#`, zwanym *kratką* albo *hashem*.
+
+Popatrzy na ten skrypt (greeting.exs):
```
Comment (kdebowski) [inline pl/lessons/basics/documentation.md]: Popatrzy - Popatrzmy

File: pl/lessons/basics/documentation.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+    iex> Greeter.hello("Sean")
+    "Hello, Sean"
+
+    iex> Greeter.hello("pete")
+    "Hello, pete"
+```
+
+### Instalacja
+
+Assuming all is well, and we're seeing the output above suggests that we are ready to set up ExDoc. Within our `mix.exs` file add the two required dependencies to get started; `:earmark` and `:ex_doc`.
```
Comment (kdebowski) [inline pl/lessons/basics/documentation.md]: Remaining english version

File: pl/lessons/basics/documentation.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+Oczywiście nie jesteśmy ograniczeni tylko do Earmarka. Jak chcesz możesz zamienić go na Pandoc, Hoedown albo Cmark; 
+to wymaga trochę dodatkowej konfiguracji, o której poczytasz [tutaj (ang.)](https://github
+.com/elixir-lang/ex_doc#changing-the-markdown-tool). Na potrzeby tej lekcji pozostaniemy jednak przy Earmarku.
+
+### Generowanie dokumentacji
+
+Idąc dalej w linii poleceń wydajemy dwie komendy:
+
+```bash
+$ mix deps.get # gets ExDoc + Earmark.
```
Comment (kdebowski) [inline pl/lessons/basics/documentation.md]: # gets ExDoc + Earmark. -> # pobiera ExDoc + Earmark.
# makes the documentation. -> # tworzy dokumentację.

I think those comments can be translatated as they are just informative for this lesson.

File: pl/lessons/basics/documentation.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+Jak widać Earmark stworzył z naszych znaczników dokumentację, a ExDoc wyświetla ją w czytelny sposób.
+
+![ExDoc Screenshot 2]({{ site.url }}/assets/documentation_2.png)
+
+Możemy ją teraz umieścić na GitHubie, naszej stronie, albo w serwisie [HexDocs](https://hexdocs.pm/).
+
+## Dobre praktyki
+
+Zasady tworzenia dokumentacji powinny być opisane w przewodniku dobrych praktyk języka. Jednakże Elixir jest jeszcze
+bardzo młodym językiem i wiele elementów jego ekosystemu musi zostać zestandaryzowane, co oznacza, że dobre praktyki
```
Comment (kdebowski) [inline pl/lessons/basics/documentation.md]: zestandaryzowane -> zestandaryzowanych

**Comment by kdebowski** [comment]
@Koziolek I've added few notes

**Comment by Koziolek** [comment]
Done

**Pull request merged**